plugin.xml

```diff
@@ -34,7 +34,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git" branch="master"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="YMChat" spec="~> 1.8" swift-version="5.0"/>
+                <pod name="YMChat" spec="~> 1.8.4" swift-version="5.0"/>
```

Comment:
Please don't target specific plugin version like this

Reply:
We have it added in `1.8.4`